src/Tests/SNIPPETS.cs

```diff
@@ -77,6 +77,21 @@ @ EntryPoint (  )
     }
 ";
 
+        public static string Op6b_Op6a =
+@"
+    /// # Summary
+    ///     This to show that the order of snippet operations is preserved:
+    operation Op6b() : Unit
+    {
+        HelloQ();
+    }
```

Comment:
Very minor suggestion here, please feel free to disregard.
```suggestion
    /// # Summary
    /// This to show that the order of snippet operations is preserved:
    operation Op6b() : Unit {
        HelloQ();
    }
```

Reply:
I agree, but I'm going to leave it, just for consistency with the whole bunch of other snippets in this file that are using this format.

Reply:
Sounds good!